tools/cldr-code/src/main/java/org/unicode/cldr/test/ExampleGenerator.java

```diff
@@ -2466,7 +2463,8 @@ private String handleDateFormatItem(String xpath, String value, boolean showCont
                     cldrFile.getWinningPath(xpath.replaceAll("dateTimeFormat", "timeFormat"));
             int tfLengthOffset = timeFormatXPathForPrefix.indexOf("timeFormatLength");
             if (tfLengthOffset < 0) {
-                return "";
+                examples.add("");
+                return;
```

Comment:
Does it make sense to add a `""` example here? or just to exit without adding anything?

Reply:
Good catch! Yes, i think it makes sense to just `return` in this case.